src/Servers/test/FunctionalTests/ServerComparison.FunctionalTests.csproj

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Server.IntegrationTesting.IIS" />
+    <ProjectReference Include="$(RepositoryRoot)src\Servers\IIS\IntegrationTesting.IIS\src\Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj" />
```

Comment:
Because it's not a shipping package?

Reply:
Correct. This assembly is a test asset, not something we ship to customers. It was only put into a package as a way to share test code between repos. That's not necessary anymore.